hvplot/util.py

```diff
@@ -79,7 +80,10 @@ def proj_to_cartopy(proj):
 
     proj = check_crs(proj)
 
-    if proj.is_latlong():
+    if hasattr(proj, 'crs'):
+        if proj.crs.is_geographic:
+            return ccrs.PlateCarree()
```

Comment:
Not looked into it but couldn't the crs be something else?

Reply:
Hmm seems from the PR on pytroll that is_geographic is meant to indicate latlong, but I'll double check

Reply:
https://github.com/pyproj4/pyproj/blob/9785bf81561ebf03446fb88d159613c4e4391f5a/pyproj/_crs.pyx#L2184

Reply:
I don't know. It is kind of frustrating. There is no mention in release notes about deprecating `is_latlong`. I guess "geographic" probably mean something very specific in this community.